temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` in committed code. Debugging statements should be removed before committing the code.
```
Remove the import statement for `pdb`.
```